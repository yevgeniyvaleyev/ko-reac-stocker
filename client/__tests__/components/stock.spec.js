import React from 'react';
import renderer from 'react-test-renderer';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
import Stock from '../../components/stock';

test('should render component', () => {
  const data = {
    id: 1,
    name: 'test',
    currentPrice: 111,
  };
  const component = renderer.create(
    <MuiThemeProvider>
      <Stock {...data} />
    </ MuiThemeProvider>,
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});

