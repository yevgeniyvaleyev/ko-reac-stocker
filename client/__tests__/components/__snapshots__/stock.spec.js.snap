// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="sc-bdVaJa jZnjlv"
  style={
    Object {
      "WebkitTapHighlightColor": "rgba(0,0,0,0)",
      "backgroundColor": "#ffffff",
      "borderRadius": 2,
      "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
      "boxSizing": "border-box",
      "color": "rgba(0, 0, 0, 0.87)",
      "fontFamily": "Roboto, sans-serif",
      "muiPrepared": true,
      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
    }
  }
>
  <span
    style={
      Object {
        "display": "inline-block",
        "width": "250px",
      }
    }
  >
    <strong>
      Name:
    </strong>
     
    <span
      className="stock-name"
    >
      test
    </span>
  </span>
  <span
    style={
      Object {
        "display": "inline-block",
        "width": "250px",
      }
    }
  >
    <strong>
      Price:
    </strong>
     
    <span
      className="stock-price"
    >
      111
    </span>
  </span>
  <a
    onClick={[Function]}
    style={
      Object {
        "float": "right",
      }
    }
  >
    Edit
  </a>
</div>
`;
